Panic on missing paramter

CentOS-83-64-minimal v0.97.2]# ./neo-go wallet nep17 transfer --wallet cn3.json -r http://seed1.neo.org:10332
Password > 
panic: address was not set

goroutine 1 [running]:
github.com/nspcc-dev/neo-go/cli/flags.(*Address).Uint160(...)
        github.com/nspcc-dev/neo-go/cli/flags/address.go:52
github.com/nspcc-dev/neo-go/cli/wallet.transferNEP(0xc00023cc60, 0x1145c31, 0x6, 0x0, 0x0)
        github.com/nspcc-dev/neo-go/cli/wallet/nep17.go:514 +0x13b5
github.com/nspcc-dev/neo-go/cli/wallet.transferNEP17(0xc00023cc60, 0x0, 0xc00017aaa0)
        github.com/nspcc-dev/neo-go/cli/wallet/nep17.go:485 +0x45
github.com/urfave/cli.HandleAction(0xf89480, 0x12b4658, 0xc00023cc60, 0xc00023cc60, 0x0)
        github.com/urfave/cli@v1.22.5/app.go:524 +0x105
github.com/urfave/cli.Command.Run(0x114925f, 0x8, 0x0, 0x0, 0x0, 0x0, 0x0, 0x11565fc, 0x15, 0x1184e19, ...)
        github.com/urfave/cli@v1.22.5/command.go:173 +0x579
github.com/urfave/cli.(*App).RunAsSubcommand(0xc000121c00, 0xc00023c9a0, 0x0, 0x0)
        github.com/urfave/cli@v1.22.5/app.go:405 +0x914
github.com/urfave/cli.Command.startApp(0x11453d6, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0x115ba99, 0x19, 0x0, ...)
        github.com/urfave/cli@v1.22.5/command.go:372 +0x7ff
github.com/urfave/cli.Command.Run(0x11453d6, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0x115ba99, 0x19, 0x0, ...)
        github.com/urfave/cli@v1.22.5/command.go:102 +0x9d4
github.com/urfave/cli.(*App).RunAsSubcommand(0xc000121a40, 0xc00023c840, 0x0, 0x0)
        github.com/urfave/cli@v1.22.5/app.go:405 +0x914
github.com/urfave/cli.Command.startApp(0x1146ce7, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0x116973c, 0x24, 0x0, ...)
        github.com/urfave/cli@v1.22.5/command.go:372 +0x7ff
github.com/urfave/cli.Command.Run(0x1146ce7, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0x116973c, 0x24, 0x0, ...)
        github.com/urfave/cli@v1.22.5/command.go:102 +0x9d4
github.com/urfave/cli.(*App).Run(0xc000121880, 0xc000132000, 0x8, 0x8, 0x0, 0x0)
        github.com/urfave/cli@v1.22.5/app.go:277 +0x808
main.main()
        command-line-arguments/main.go:19 +0x4e

--- a/cli/nep17_test.go
+++ b/cli/nep17_test.go
@@ -116,6 +116,14 @@ func TestNEP17Transfer(t *testing.T) {
 		"--from", validatorAddr,
 	}
 
+	t.Run("missing receiver", func(t *testing.T) {
+		as := append([]string{}, args[:8]...)
+		as = append(as, args[10:]...)
+		e.In.WriteString("one\r")
+		e.RunWithError(t, as...)
+		e.In.Reset()
+	})
+
 	t.Run("InvalidPassword", func(t *testing.T) {
 		e.In.WriteString("onetwothree\r")
 		e.RunWithError(t, args...)

--- a/cli/wallet/nep17.go
+++ b/cli/wallet/nep17.go
@@ -514,6 +514,9 @@ func transferNEP(ctx *cli.Context, standard string) error {
 	}
 
 	toFlag := ctx.Generic("to").(*flags.Address)
+	if !toFlag.IsSet {
+		return cli.NewExitError(errors.New("missing receiver address (--to)"), 1)
+	}
 	to := toFlag.Uint160()
 	token, err := getMatchingToken(ctx, wall, ctx.String("token"), standard)
 	if err != nil {

